tracing/test_static_max_level_features/tests/test.rs

```diff
@@ -33,7 +33,7 @@ impl Subscriber for TestSubscriber {
     fn exit(&self, _span: &Id) {}
 }
 
-#[cfg(test)]
+#[test]
```

Comment:
this test was never actually running :upside_down_face:

Reply:
RIP